tests/e2e/wdio.conf.js

```diff
@@ -0,0 +1,27 @@
+import path from 'node:path';
```

Comment:
why don't we put this file in the project root?

Reply:
The debug version is so different, that it is clearer to have two saparate configurations. Because of that we have two files, and this is only required for running end-to-end tests, so I thought that keeping it nested is clearer.